Cubical/Categories/Instances/PosetCat/MonotoneAdjoint.agda

```diff
@@ -0,0 +1,105 @@
+{-# OPTIONS --safe #-}
+
+-- defines adjoint for monotone functions and morphisms in the Adjoint category
+
+module Cubical.Categories.Instances.PosetCat.MonotoneAdjoint where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.HLevels
+
+open import Cubical.Relation.Binary.Poset
+open import Cubical.Data.Sigma
+open import Cubical.HITs.PropositionalTruncation.Base
+open import Cubical.HITs.PropositionalTruncation.Properties
+
+open import Cubical.Categories.Instances.PosetCat.Monotone
+
+
+private
+  variable
+    ℓ ℓ' : Level
+
+
+module _ {ℓ ℓ' : Level} where
+
+  -- adjoints for monotone functions
```

Comment:
Also called Galois connections. Should at least mention this in a comment for searchability